example/data-migration-ecs/main.tf

```diff
@@ -0,0 +1,30 @@
+terraform {
+  required_providers {
+    aws  = ">= 3.5.0"
+  }
+}
+
+provider "aws" {
+  region  = var.region
+  ignore_tags {
+    key_prefixes = ["gsfc-ngap"]
+  }
+}
+
+module "data_migration_ecs_service" {
+  source = "https://github.com/nasa/cumulus/releases/download/v4.0.0/terraform-aws-cumulus-ecs-service.zip"
+
+  prefix = var.prefix
+  name   = "ExecutionMigrationService"
+
+  log2elasticsearch_lambda_function_arn = var.log2elasticsearch_lambda_function_arn
+  cluster_arn                           = var.ecs_cluster_arn
```

Comment:
These can probably be taken from the cumulus remote state.

Reply:
👍🏻  for /example use lookups where possible.